components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -1875,8 +1881,15 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                         }
                         fields.put(subParts[0].replace(":", "_"), value);
                     } else {
-                        String value = subParts[1];
-                        if(value.contains(" ")) {
+                        String value = subParts[1].toLowerCase();
+
+                        if(value.contains(" or ")){
+                            String[] values = value.split(" or ");
+                            for(int i=0; i<values.length; i++){
```

Comment:
Thanks for pointing it out. It is fixed with https://github.com/wso2/carbon-governance/pull/239